src/Feed/EventdatabasenFeedType.php

```diff
@@ -0,0 +1,246 @@
+<?php
+
+namespace App\Feed;
+
+use App\Entity\Feed;
+use App\Entity\FeedSource;
+use App\Service\FeedService;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * See https://github.com/itk-event-database/event-database-api.
+ */
+class EventdatabasenFeedType implements FeedTypeInterface
```

Comment:
"Eventdatabasen" should properply be shange to "DetSkerIAarhus", which is the official name: https://api.detskeriaarhus.dk/

*  not a mix between danish and english
* scopes it to Aarhus, which we should do when putting Aarhus specific things in "core"

Reply:
Changed to EventDatabaseApiFeedType